modules/core/src/v2/coins/eth.ts

```diff
@@ -1674,16 +1675,23 @@ export class Eth extends BaseCoin {
     if (coinSpecific.forwarderVersion === 0) {
       return true;
     } else {
-      const forwarderFactoryAddress = config.defaults[self.getChain()].forwarderFactoryAddress;
-      const initcode = Util.getProxyInitcode(config.defaults[self.getChain()].forwarderImplementationAddress);
+      const forwarderFactoryAddress = (self.staticsCoin?.network as EthereumNetwork).forwarderFactoryAddress!;
```

Comment:
Can we turn this into a method so we don't have to cast every time

`(self.staticsCoin?.network as EthereumNetwork)`

--->

```
getNetwork(): EthereumNetwork | undefined {
   ...
}
```

Reply:
Thanks for the suggestion. I have updated the PR